client/webserver/site/src/js/wallets.js

```diff
@@ -373,6 +373,7 @@ export default class WalletsPage extends BasePage {
       app().notify(ntfn.make(`No wallet found for ${asset.info.name}`, 'Cannot withdraw.', ntfn.ERROR))
     }
     await this.hideBox()
+    page.asset = asset
```

Comment:
`page` is for html elements parsed from the loaded page. Just use `this`, in the same way that we do `this.openAsset` for the unlock form, or `this.reconfigAsset` for the reconfigure form. Maybe `this.withdrawAsset`?